sdk/tests/conformance2/misc/uninitialized-test-2.html

```diff
@@ -179,127 +179,390 @@
+      },
+      {
+        target: "TEXTURE_3D",
+        internal_format: "RGBA8UI",
+        format: gl.RGBA_INTEGER,
+        type: gl.UNSIGNED_BYTE,
+        read_type: gl.UNSIGNED_INT,
+        width: 256, // minimum MAX_3D_TEXTURE_SIZE is 256
+        height: 256,
+        depth: 8,
+        exceptions: [],
+      },
+      {
+        target: "TEXTURE_3D",
+        internal_format: "RGBA8UI",
+        format: gl.RGBA_INTEGER,
+        type: gl.UNSIGNED_BYTE,
+        read_type: gl.UNSIGNED_INT,
+        width: max_3d_texture_size,
+        height: max_3d_texture_size,
+        depth: 4,
+        exceptions: [ { x: 128, y: 0, r: 108, g: 72, b: 36, a: 9 } ],
+      },
+      {
+        target: "TEXTURE_3D",
+        internal_format: "RGBA8UI",
+        format: gl.RGBA_INTEGER,
+        type: gl.UNSIGNED_BYTE,
+        read_type: gl.UNSIGNED_INT,
+        width: max_3d_texture_size,
+        height: max_3d_texture_size,
+        depth: 4,
+        exceptions: [],
+      },
 
-// The minimum max_texture_3d is 256.
-width = 256;
-height = 256;
-depth = 8;
+      // TEXTURE_3D + RGBA8I
+      {
+        target: "TEXTURE_3D",
+        internal_format: "RGBA8I",
+        format: gl.RGBA_INTEGER,
+        type: gl.BYTE,
+        read_type: gl.INT,
+        width: 256, // minimum MAX_3D_TEXTURE_SIZE is 256
+        height: 256,
+        depth: 8,
+        exceptions: [ { x: 128, y: 255, r: 108, g: 72, b: 36, a: 9 } ],
+      },
+      {
+        target: "TEXTURE_3D",
+        internal_format: "RGBA8I",
+        format: gl.RGBA_INTEGER,
+        type: gl.BYTE,
+        read_type: gl.INT,
+        width: 256, // minimum MAX_3D_TEXTURE_SIZE is 256
+        height: 256,
+        depth: 8,
+        exceptions: [],
+      },
+      {
+        target: "TEXTURE_3D",
+        internal_format: "RGBA8I",
+        format: gl.RGBA_INTEGER,
+        type: gl.BYTE,
+        read_type: gl.INT,
+        width: max_3d_texture_size,
+        height: max_3d_texture_size,
+        depth: 4,
+        exceptions: [ { x: 128, y: 128, r: 108, g: 72, b: 36, a: 9 } ],
+      },
+      {
+        target: "TEXTURE_3D",
+        internal_format: "RGBA8I",
+        format: gl.RGBA_INTEGER,
+        type: gl.BYTE,
+        read_type: gl.INT,
+        width: max_3d_texture_size,
+        height: max_3d_texture_size,
+        depth: 4,
+        exceptions: [],
+      },
 
-targets = [ "TEXTURE_3D", "TEXTURE_2D_ARRAY" ];
-var internal_formats = [ "RGBA8", "RGBA8UI", "RGBA8I" ];
-var formats = [ gl.RGBA, gl.RGBA_INTEGER, gl.RGBA_INTEGER ];
-var types = [ gl.UNSIGNED_BYTE, gl.UNSIGNED_BYTE, gl.BYTE ];
-var read_types = [ gl.UNSIGNED_BYTE, gl.UNSIGNED_INT, gl.INT ];
+      // TEXTURE_2D_ARRAY + RGBA8
+      {
+        target: "TEXTURE_2D_ARRAY",
+        internal_format: "RGBA8",
+        format: gl.RGBA,
+        type: gl.UNSIGNED_BYTE,
+        read_type: gl.UNSIGNED_BYTE,
+        width: 2048,
+        height: 2048,
```

Comment:
Would it be possible to reproduce the same issues with a smaller width/height for these 2D texture array cases, like 1024x1024? If so please adjust the test cases. Thanks.